app/controllers/orders_controller.rb

```diff
@@ -39,7 +53,12 @@ def destroy
     end
 
     def index
-        @orders = current_user.orders
+        begin
+            authorize current_user, :isAdmin?
+            @orders = Order.all
+        rescue Pundit::NotAuthorizedError
+            @orders = current_user.orders
```

Comment:
This method does not look good.

have it like 
```
if current_user.admin?
  @orders = Order.all
else
  @orders = current_user.orders
end
```
and in orders_policy.rb
```
def index?
  admin? || customer?
end

```

Reply:
def index?
  admin? || customer?
end

this will always return true